src/main/java/com/cascadebot/cascadebot/permissions/Permission.java

```diff
@@ -5,25 +5,27 @@
 
 package com.cascadebot.cascadebot.permissions;
 
-import com.cascadebot.cascadebot.commandmeta.CommandType;
+import com.cascadebot.cascadebot.commandmeta.Module;
 
 import java.util.Arrays;
 import java.util.EnumSet;
 
 public class Permission {
```

Comment:
We might want to rename our permission to like "CascadPermission" so we don't have problmes with jda permissions.